### ISSUE TITLE
Eslint 9

### ISSUE THREAD

**Issue by steida** [evoluhq/evolu#392]
Add it to CI

**Comment by steida** [comment]
Blocked by ecosystem readiness.

**Comment by steida** [comment]
https://x.com/karlhorky/status/1792500811307622631?s=46&t=1GScoXvy3XhWLnbkX-GC5g

**Comment by needim** [comment]
Alternative: Biomejs

**Comment by steida** [comment]
@needim I don't think it's viable. For example, React Eslint, we need that for deps.

**Comment by steida** [comment]
Closing as this is no longer relevant—the new Evolu launches tomorrow. Feel free to reopen if it becomes relevant with the new Evolu.
